dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1720,3 +1726,54 @@ def test_std_raises_on_index():
         match="`std` is only supported with objects that are Dataframes or Series",
     ):
         dd.from_pandas(pd.DataFrame({"test": [1, 2]}), npartitions=2).index.std()
+
+
+@pytest.mark.skipif(not PANDAS_GT_200, reason="ArrowDtype not supported")
+def test_std_raises_with_arrow_string_ea():
+    pa = pytest.importorskip("pyarrow")
+    ser = pd.Series(["a", "b", "c"], dtype=pd.ArrowDtype(pa.string()))
+    ds = dd.from_pandas(ser, npartitions=1)
+    with pytest.raises(ValueError, match="`std` not supported with string series"):
+        ds.std()
+
+
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        pytest.param(
+            "int64[pyarrow]",
+            marks=pytest.mark.skipif(
+                pa is None or not PANDAS_GT_150, reason="requires pyarrow installed"
+            ),
+        ),
+        pytest.param(
+            "float64[pyarrow]",
+            marks=pytest.mark.skipif(
+                pa is None or not PANDAS_GT_150, reason="requires pyarrow installed"
+            ),
+        ),
+        "Int64",
+        "Int32",
+        "Float64",
+        "UInt64",
+    ],
+)
+@pytest.mark.parametrize("func", ["std", "var", "skew", "kurtosis"])
+def test_reductions_with_pandas_and_arrow_ea(dtype, func):
+    if func in ["skew", "kurtosis"]:
+        if not scipy:
+            pytest.skip()
+        elif "pyarrow" in dtype:
+            pytest.xfail("skew/kurtosis not implemented for arrow dtypes")
+
+    ser = pd.Series([1, 2, 3, 4], dtype=dtype)
+    ds = dd.from_pandas(ser, npartitions=1)
```

Comment:
Similar comment here re: `npartitions`

```suggestion
    ds = dd.from_pandas(ser, npartitions=2)
```

Reply:
Done